snapcraft/tests/test_yaml.py

```diff
@@ -656,22 +653,11 @@ def test_invalid_yaml_invalid_epochs(self, mock_loadPlugin):
                 with self.assertRaises(SnapcraftSchemaError) as raised:
                     internal_yaml.Config()
 
-                if 'less than' in raised.exception.message:
-                    self.assertRegex(
-                        raised.exception.message,
-                        "The 'epoch' property does not match the required "
-                        "schema:.*is less than the minimum of 0")
-                elif 'not a multiple' in raised.exception.message:
-                    self.assertRegex(
-                        raised.exception.message,
-                        "The 'epoch' property does not match the required "
-                        "schema:.*is not a multiple of 1.0")
-                else:
-                    self.assertRegex(
-                        raised.exception.message,
-                        "The 'epoch' property does not match the required "
-                        "schema:.*does not match '{}'".format(
-                            re.escape('^(?:0|[1-9][0-9]*[*]?)$')))
+                self.assertRegex(
+                    raised.exception.message,
+                    "The 'epoch' property does not match the required "
+                    "schema:.*is not a 'epoch' \(epochs are positive integers "
```

Comment:
It's actually hard-coded in jsonschema to 'is not a'. Terrible, eh? But I wanted to use the format checkers the way they intended...